CONTRIBUTING.md

```diff
@@ -1,17 +1,183 @@
-# Contributing to LLVM
+# Contributing to the Arm Toolchain
 
-Thank you for your interest in contributing to LLVM! There are many ways to
-contribute, and we appreciate all contributions.
+The Arm Toolchain repository is a fork of the *llvm-project*, with
+build scripts to produce Arm Toolchain for Embedded and Arm Toolchain
+for Linux.
 
-To get started with contributing, please take a look at the
-[Contributing to LLVM](https://llvm.org/docs/Contributing.html) guide. It
-describes how to get involved, raise issues and submit patches.
+Arm Toolchain for Embedded integrates *llvm-project* with the
```

Comment:
Do we need to mention llvm-libc separately as well? It is part of llvm-project so implicitly is there, however we seem to go into much more details in regard to the other two libraries.

Taking into account that we want to use llvm-libc more, we may want to mention is explicitly.

Reply:
The distinction I was trying to make was in-tree llvm-project, which includes llvm-libc, and out-of-tree picolibc and newlib. LLVM libc

I've updated the wording to hopefully make it clearer.